canonical-websites/www.ubuntu.com
# remove pre-v1 content, rename -v1 conent

## Done

* remove all pre Vanilla v1 pages
* remove ‘-v1’ from all directories and file names
* remove ‘-v1’ from all html and scss files

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- look at the homepage and at least one page from every section
- try searching
- try filling out a form
- try making a contribution
- see the site looks ok

## Note

* we don’t seem to have any ie.css files or print.css is that correct?
* are there more pages to delete?
